### ISSUE TITLE
Emitting active question can fail for issues with secret voting

### ISSUE THREAD

**Issue by duvholt** [dotkom/super-duper-fiesta#292]
Steps to reproduce:

 - Login in as admin without completing registration
 - Open issue with secret voting
 - Refresh page

```
error: Getting currently active issue failed. TypeError: Cannot read property 'id' of null
    at emitActiveQuestion (/home/christian/Projects/super-duper-fiesta/server/channels/connection.js:88:52)
    at <anonymous>
```

The problem is that the voter object is none, but we still try to access voter.id.

**Comment by sklirg** [comment]
Just adding some context:

It happens because we try to retrieve the vote for a user when they load the webpage (which happens after a page reload) so that they can see their vote in the client. However, if they haven't completed the registration, they don't have an anonymousUser, which will fail the getAnonUserVote function. As an administrator you can actually bypass registration _when accessing the admin page_, but you can not vote on issues from the admin page.

This can actually be fixed by implementing #289, but a simpler fix is probably possible. We could for example try to separate user and admin channels, so that we don't emit all actions to both when it's not needed, or simply check if the user is registered before trying to fetch their votes.

**Comment by duvholt** [comment]
This error also triggers if a user tries to register while an issue with secret voting is active during registration.
